chart/templates/scheduler/scheduler-service.yaml

```diff
@@ -18,7 +18,7 @@
 ################################
 ## Airflow Scheduler Service
 #################################
-{{- if eq .Values.executor "LocalExecutor" }}
+{{- if or (eq .Values.executor "LocalExecutor") (eq .Values.executor "LocalKubernetesExecutor")}}
```

Comment:
I think if we want to make our Chart fully compatible with AIP-51 and allow "any" executors we should also implement some way to override Helm Chart behaviour for custom executors. But we can do it later. CC: @o-nikolas

Reply:
Ah.. I see @dstandish already had some comments on that one too :) Yeah - serve_logs should be fine and it should have default value depending on the "built-in" executors - so that we do not have to configure it manually for built-in executors.